### ISSUE TITLE
Modeless dialog

### ISSUE THREAD

**Issue by yong2579** [nakupanda/bootstrap3-dialog#227]
Anyone think having the option to choose whether a dialog is modal or modeless would be good? I have customize this project with the mentioned and if anyone especially the author is interested, I would like to contribute to this project as an enhancement.

**Comment by nakupanda** [comment]
Interesting, any online demo to be shown at this moment?

**Comment by yong2579** [comment]
Thanks for the interest dante, I had work up a demo @ http://yong2579.github.io/ but realise the customization has 2 flaws:

1) missing scroll bar when page is scrollable
2) will see a flickering backdrop if animate is set to true

So basically its more of a demo rather than contribution until I look into the appropriate fix

**Comment by nakupanda** [comment]
Modaless is a cool idea as some people had asked for similar before
https://github.com/nakupanda/bootstrap3-dialog/issues/178

There are more but I can't recall at this moment.

**Comment by yong2579** [comment]
Yeah it's a shame that bootstrap dialog does not come in built with the feature unlike jquery ui dialog

**Comment by yong2579** [comment]
Demo updated, fixes include:

1) scroll bar will not go missing after opening dialog
2) no flickering backdrop if animate is set to true

However there is one more item that doesn't seems fixable

1) scrollbar will disappear during the animation period and reappear after the animation end

Additionally it is discovered that the dragging of modeless dialog seems to accidentally select the background and lost the dragging, probably need to explore a little more into the drag feature

**Comment by yong2579** [comment]
Manage to fix the lost dragging handle but part of the solution include having to remove the mouseleave event handler. Hmm not too sure why it is initially required, can shed some light?

                this.getModal().on('mouseup mouseleave', {dialog: this}, function (event) {
                    event.data.dialog.draggableData.isMouseDown = false;

**Comment by TimWoosh** [comment]
Hi, I'm also interested in modeless dialogs

Are there any plans to include this functionality in bootstrap3-dialog?

**Comment by nakupanda** [comment]
Hi @yong2579 ,

I'd like to see your implementation of the modaless option, where is it?

**Comment by nakupanda** [comment]
I read some from http://yong2579.github.io/src/js/bootstrap-dialog.js thanks

**Comment by alligator77** [comment]
this enhancement implemented in release?
